test/gc/test-net-timeout.js

```diff
@@ -30,14 +29,14 @@ let countGC = 0;
 console.log('We should do ' + todo + ' requests');
 
 var server = net.createServer(serverHandler);
-server.listen(PORT, getall);
+server.listen(0, getall);
 
 function getall() {
   if (count >= todo)
     return;
 
   (function() {
-    var req = net.connect(PORT, '127.0.0.1');
+    var req = net.connect(server.address().port, '127.0.0.1');
```

Comment:
While you're here, can you remove the bind address?  If the server binds to ::1, connecting to 127.0.0.1 doesn't have to work.